css/menu.css

```diff
@@ -110,17 +142,16 @@ li a:hover:not(.active) {
   }
 
   .modal {
-      padding-top: 0;
+    padding-top: 0;
   }
 
   .lg {
     display: none;
   }
 }
-
 /* The Close Button */
 .closeBtn {
-  color: #aaaaaa;
+  color: #aaa;
```

Comment:
🤔 I think the standard is to have 6 chars (HEX)

Reply:
Unfortunately, this is due to Sass doing that, the un-compiled version does not have it.